device/marvell/armhf-marvell_et6448m_52x-r0/syncd.conf

```diff
@@ -0,0 +1,5 @@
+sysctl -w net.core.rmem_max=509430500
+sysctl -w net.core.rmem_default=2097152
+sysctl -w net.ipv4.neigh.default.gc_thresh1=16000
+sysctl -w net.ipv4.neigh.default.gc_thresh2=32000
+sysctl -w net.ipv4.neigh.default.gc_thresh3=48000
```

Comment:
are they specific to marvell, they are generic sonic setting, not sure why you need them for this sku.

Reply:
kernel had default values as 128, 512 and 1024 resp. So we had to follow as per
https://github.com/Azure/SONiC/pull/399/files (Line 92)

Reply:
let them handle it? this should not be platform specific.

Reply:
Done. Removed the config file